packages/mdc-textfield/constants.js

```diff
@@ -41,6 +41,7 @@ const cssClasses = {
   TEXTAREA: 'mdc-text-field--textarea',
   OUTLINED: 'mdc-text-field--outlined',
   WITH_LEADING_ICON: 'mdc-text-field--with-leading-icon',
+  HELPER_LINE: 'mdc-text-field-helper-line',
```

Comment:
I discussed this with Abhinay offline. One important aspect of the containing element is to establish flexbox layout rather than needing to resort to floated elements.